### ISSUE TITLE
looks for missing json schema in current rustc output

### ISSUE THREAD

**Issue by nergdron** [AtomLinter/linter-rust#163]
Fresh install of rust via rustup, and of this extension. It fails on a simple hello world main.rs, with a visible error in the gui reading:

```
Failed to run rustc with exit code 1
with args:
 --cfg test --color never /home/tessa/code/mine/rusty/main.rs --error-format=json
See console for more information
```

looking in the js console log, I see the output from rustc being printed correctly, but then this javascript error:

```js
Error setting item in object: Validation failed at linter-rust.disabledWarnings, undefined must be an array
coerce @ <embedded>:11
```

looking at the output of running the command manually, this is the data returned:

```json
{"message":"cannot find value `dsgasdgds` in this scope","code":{"code":"E0425","explanation":"An unresolved name was used.\n\nErroneous code examples:\n\n```compile_fail,E0425\nsomething_that_doesnt_exist::foo;\n// error: unresolved name `something_that_doesnt_exist::foo`\n\n// or:\n\ntrait Foo {\n    fn bar() {\n        Self; // error: unresolved name `Self`\n    }\n}\n\n// or:\n\nlet x = unknown_variable;  // error: unresolved name `unknown_variable`\n```\n\nPlease verify that the name wasn't misspelled and ensure that the\nidentifier being referred to is valid for the given situation. Example:\n\n```\nenum something_that_does_exist {\n    Foo,\n}\n```\n\nOr:\n\n```\nmod something_that_does_exist {\n    pub static foo : i32 = 0i32;\n}\n\nsomething_that_does_exist::foo; // ok!\n```\n\nOr:\n\n```\nlet unknown_variable = 12u32;\nlet x = unknown_variable; // ok!\n```\n\nIf the item is not defined in the current module, it must be imported using a\n`use` statement, like so:\n\n```\n# mod foo { pub fn bar() {} }\n# fn main() {\nuse foo::bar;\nbar();\n# }\n```\n\nIf the item you are importing is not defined in some super-module of the\ncurrent module, then it must also be declared as public (e.g., `pub fn`).\n"},"level":"error","spans":[{"file_name":"/home/tessa/code/mine/rusty/main.rs","byte_start":47,"byte_end":56,"line_start":3,"line_end":3,"column_start":5,"column_end":14,"is_primary":true,"text":[{"text":"    dsgasdgds","highlight_start":5,"highlight_end":14}],"label":"not found in this scope","suggested_replacement":null,"suggestion_applicability":null,"expansion":null}],"children":[],"rendered":"error[E0425]: cannot find value `dsgasdgds` in this scope\n --> /home/tessa/code/mine/rusty/main.rs:3:5\n  |\n3 |     dsgasdgds\n  |     ^^^^^^^^^ not found in this scope\n\n"}
{"message":"aborting due to previous error","code":null,"level":"error","spans":[],"children":[],"rendered":"error: aborting due to previous error\n\n"}
{"message":"For more information about this error, try `rustc --explain E0425`.","code":null,"level":"failure-note","spans":[],"children":[],"rendered":"For more information about this error, try `rustc --explain E0425`.\n"}
```

this all looks correct, so it feels like the linter extension isn't correctly interpreting current rustc output.

**Comment by nergdron** [comment]
Blew away my atom config, and with a fresh atom install, I can't repro this problem. so it must've been because of some other broken package in my install.
